robot_skills/src/robot_skills/arms.py

```diff
@@ -597,13 +647,14 @@ def occupied_by(self, value):
         """
         self._occupied_by = value
 
-    def send_gripper_goal(self, state, timeout=5.0):
+    def send_gripper_goal(self, state, timeout=5.0, max_torque=0.1):
         """
         Send a GripperCommand to the gripper of this arm and wait for finishing
         :param state: open or close
         :type state: str (GripperState)
         :param timeout: timeout in seconds; timeout of 0.0 is not allowed
         :type timeout: float
+        :param max_torque: How much force to apply, range between 0 and 1, only applied if closing the gripper
```

Comment:
Torque/force, please be consistent. Also the range between 0 and 1 is very unclear. Please provide units.